examples/vsphere/sample-terraform-dhcp.tfvars

```diff
@@ -99,21 +99,20 @@ node_pools = {
   system = {
     count       = 1
     cpus        = 8
-    memory      = 16384
+    memory      = 65536
     os_disk     = 100
     node_taints = []
     node_labels = {
       "kubernetes.azure.com/mode" = "system" # REQUIRED LABEL - DO NOT REMOVE
     }
   },
   cas = {
-    count   = 3
+    count   = 1
     cpus    = 16
     memory  = 131072
     os_disk = 350
     misc_disks = [
-      150,
-      150,
+      400,
```

Comment:
Be careful here. The 2 different disk are used for 2 different items. Though they look the same they are needed separately.

Reply:
I'll message you about this, there may be something about the `misc_disks` that I'm not aware of.